sdk/file_sdk/Common/Common.pro

```diff
@@ -4,7 +4,13 @@ TARGET    = modfilecommon
 
 TEMPLATE  = lib
 CONFIG += staticlib warn_on c++11 debug_and_release
+
 INCLUDEPATH += $$REPO_ROOT/sdk/file_sdk
+win32:INCLUDEPATH += $$REPO_ROOT/third_party/include/Libgsf
+
+win32:LIBS += -L$$REPO_ROOT/third_party/lib/Libgsf -llibgsf-1-114 -lzlib1\
+               -llibgobject-2.0-0\
```

Comment:
I don't think you need it here, only in the tests

Reply:
I need this for LibGsf, Gsf_input_istream.cpp

---
In reply to: [139357750](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139357750) [](ancestors = 139357750)

Reply:
I know, but it is a static lib,  only the unit tests are actually using it.
I may be wrong but i think that everything will compile if you remove it

---
In reply to: [139372975](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139372975) [](ancestors = 139372975,139357750)